### PR TITLE
chore: update condition

### PR DESCRIPTION
**Pull request by devceline**
# changes
- use `isInitialSubscriptionLoadComplete` for more accurate status

### CODE REVIEW

File: src/w3iProxy/notifyProviders/internalNotifyProvider.ts [hunk 1]
```diff
@@ -106,8 +103,8 @@ export default class InternalNotifyProvider implements W3iNotifyProvider {
 
   // ------------------- Method-forwarding for NotifyClient -------------------
 
-  public isInitialSubscriptionLoadComplete() {
-    return this.gotFirstWatchSubscriptionsResponse;
+  public hasFinishedInitialLoad() {
+    return this.notifyClient?.hasFinishedInitialLoad() || false;
```
Comment (enesozturk) [inline src/w3iProxy/notifyProviders/internalNotifyProvider.ts:107]: Do we need `|| false` here? If yes, do we have a case that `this.notifyClient?.hasFinishedInitialLoad()` always falsy?

Comment (devceline) [reply src/w3iProxy/notifyProviders/internalNotifyProvider.ts:107]: just added the `|| false` instead `notifyClient` is `undefined`

Comment (enesozturk) [reply src/w3iProxy/notifyProviders/internalNotifyProvider.ts:107]: @devceline I mean that on the Notify side, if somehow it's not set to true, we'll still show skeletons infinitely, right?

Comment (devceline) [reply src/w3iProxy/notifyProviders/internalNotifyProvider.ts:107]: Yes, but that should never ever happen. We set this to true even if there an error

Comment (enesozturk) [reply src/w3iProxy/notifyProviders/internalNotifyProvider.ts:107]: Understood. We should always handle all cases. Wanted to double check

**Review by enesozturk — APPROVED** [review]

**Pull request merged**